### ISSUE TITLE
Looking for collaborators

### ISSUE THREAD

**Issue by Schniz** [Schniz/render.rs#20]
Hi there!

Unfortunately, I don't use Rust as much as I would want.
If some of you use this library and would like to get a push access to make it better, let me know.

**Comment by Schniz** [comment]
cc @theashguy @vpzomtrrfrt 😄

**Comment by theashguy** [comment]
Yup. Definitely keen! Have been meaning to reach out to chat! :).

**Comment by theashguy** [comment]
Can we set up a github org, and maybe a discord channel for the project so we can manage things better / get in sync? 

I've also may have got super excited and gone ahead and purchased http://render.rs...

![Screen Shot 2020-06-11 at 10 34 46 am](https://user-images.githubusercontent.com/744872/84332073-486e7580-abcf-11ea-8cc0-32b25495e6fa.png)

**Comment by vpzomtrrfrt** [comment]
I would personally prefer Matrix, but Discord works

**Comment by theashguy** [comment]
Whatever is good. I just feel like everyone is coalescing inside Discord these days 🤷

**Comment by Schniz** [comment]
Personally, I like both although I haven't _really_ used Matrix, just like the idea of using an open source tech. I'm on the Reason Discord server so I kinda have my preference but I'd suggest more options to consider 😃 

* For real time chats, [Gitter](https://gitter.im) would be easier to "just hop on to". It's far less exciting than Discord or Matrix but it gets the job done, and you don't need to log in or have a user to read chats. You don't need to install stuff. It can also be hooked up to Matrix (AFAIK) and using [`matterbridge`](https://github.com/42wim/matterbridge) it can also be bridged to Discord or other services.
* I would also suggest thinking about something which is less real-time so more people around the world can join the "meaningful" conversations and have more context. I'm GMT+02, I guess some of you are in SF time zone (a 10 hours diff). So something like [Spectrum](https://spectrum.chat) would _maybe_ be a better fit here?

  The biggest issue with Spectrum is that I'm not entirely sure what's the current status since they got acquired by GitHub and the release of GitHub Discussions.

I think that Gitter + Spectrum/GH Discussions would make the most sense to me. What do you think?

(Sorry for the bikeshedding 😅)

**Comment by daaku** [comment]
> I would also suggest thinking about something which is less real-time so more people around the world can join the "meaningful" conversations and have more context. I'm GMT+02, I guess some of you are in SF time zone (a 10 hours diff). So something like Spectrum would maybe be a better fit here?

+1 to preferring asynchronous communication, and GitHub conversations are already part of the workflow.

**Comment by vpzomtrrfrt** [comment]
Yeah, at this scale I'm not sure if a chat platform is necessary compared to GitHub issues

**Comment by theashguy** [comment]
Yah, the only problem with GitHub issues is if / when core team members need fast direct comms with one another (ie, when setting a bunch of this stuff up and frankly, getting used to each other :p). Otherwise I'm all for everything being documented and run through GitHub :+1:.

**Comment by srid** [comment]
Hey - this library looks interesting, but hasn't been updated since summer last year. Does anyone maintain an active fork, or is there somebody that's interested in taking over maintainership?

**Comment by Schniz** [comment]
Hey, as I wrote earlier. I don’t use Rust for servers as much as I wanted to so it’s hard to maintain it (I have no real use cases). I will try to release the main branch soon though.

**Comment by theashguy** [comment]
I've been working to transition one of my projects to this-- it still _seems_ to be the least restrictive solution out there for now (there are other things in this space but they all have show-stopper issues, whereas all of renders issues can be relatively easily worked around).  I do maintain a fork that I work from but I don't think there is anything new in there.

**Comment by thomaseizinger** [comment]
What is the status on this? https://github.com/orgs/render-rs/people doesn't show any public members.

I am successfully using this library and happy to push it forward a bit (e.g. https://github.com/render-rs/render.rs/issues/49) but would like to confirm with some of the existing maintainers.

**Comment by brahmlower** [comment]
Hey folks, similar to @thomaseizinger's comment, I'm curious if a group was ever set up? I've been really enjoying the simplicity of this library and would love to help keep it going. I wouldn't be able to drive it full time or anything, but could dedicate at least a couple hours a week to maintenance with the occasional weekend sprint 😁 

edit: regarding the above discussion about team coordination- if a communication channel hasn't already been decided on, github has private discussions for org teams that could be used 🤷

**Comment by AustinHellerRepo** [comment]
I've been contributing to https://github.com/pyrossh/tide-jsx in the absence of activity on this repo. Please consider pulling in my PRs from that repo to fix the script tag and make the `raw` macro more convenient.

**Comment by DougAnderson444** [comment]
+1 Can we bump this issue and get some additional maintainers on the access list to merge pull requests?
